blackbox/docs/admin/ssl.txt

```diff
@@ -285,6 +285,15 @@ The common name (CN) should overlap with the CN of the server key
 generated in the first step. For example, ``ssl.crate.io`` overlaps
 with `*.crate.io`.
 
+.. NOTE::
+
+    In this step by step guide it is shown how to create a server certificate.
+    If you want to create a client certificate the steps are the same with the
+    exception of providing a common name that is equivalent to the crate
+    username as described here: `client certificate authentication method`_.
```

Comment:
I wouldn't keep `here` and the link.
Would either say `...as describe in: <link>` or `as described here` the `here` is already the link